subxt/src/book/usage/transactions.rs

```diff
@@ -77,7 +77,6 @@
 //! Let's see how to use each of these approaches:
```

Comment:
I think here I'd be inclined to only show how to do (1), and then link to your example for (2) since it's suddenly quite verbose